sound/soc/intel/boards/sof_sdw_rt_dmic.c

```diff
@@ -47,6 +51,17 @@ int rt_dmic_rtd_init(struct snd_soc_pcm_runtime *rtd)
 	if (!card->components)
 		return -ENOMEM;
 
+	codec_dai = get_codec_dai_by_name(rtd, multi_function_dmics,
+					  ARRAY_SIZE(multi_function_dmics));
+	if (codec_dai) {
```

Comment:
you could just run the match against the previously found codec_dai->name, right?

Reply:
> you could just run the match against the previously found codec_dai->name, right?

Indeed, thanks for the suggestion.